scripts/auto-rebase/rebase.sh

```diff
@@ -1118,6 +1119,44 @@ EOF
     done  # for goarch
 }
 
+update_kubeproxy_images() {
```

Comment:
If kubeproxy is only used with flannel, should we move this code to the dedicated script too?

Reply:
I didn't move this in separate script because kube-proxy is core component of release artifacts so doesn't make sense to pull that artifacts again and parse it. It is optional like olm but part of release artifacts so better to be here ?

Reply:
OK, if there's an analogy with other existing components - that's fine.